soc/nxp/imx/imx95/a55/mmu_regions.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2022 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/arch/arm64/arm_mmu.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/sys/util.h>
+
+static const struct arm_mmu_region mmu_regions[] = {
+
+	MMU_REGION_FLAT_ENTRY("GIC",
+			      DT_REG_ADDR_BY_IDX(DT_NODELABEL(gic), 0),
+			      DT_REG_SIZE_BY_IDX(DT_NODELABEL(gic), 0),
+			      MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_NS),
+
+	MMU_REGION_FLAT_ENTRY("GIC",
+			      DT_REG_ADDR_BY_IDX(DT_NODELABEL(gic), 1),
+			      DT_REG_SIZE_BY_IDX(DT_NODELABEL(gic), 1),
+			      MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_NS),
+
+#ifndef CONFIG_SOC_MIMX9596_A55
+	MMU_REGION_FLAT_ENTRY("CCM",
+			      DT_REG_ADDR(DT_NODELABEL(ccm)),
+			      DT_REG_SIZE(DT_NODELABEL(ccm)),
+			      MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_NS),
+
+	MMU_REGION_FLAT_ENTRY("ANA_PLL",
+			      DT_REG_ADDR(DT_NODELABEL(ana_pll)),
+			      DT_REG_SIZE(DT_NODELABEL(ana_pll)),
+			      MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_NS),
+
+	MMU_REGION_FLAT_ENTRY("IOMUXC",
+			      DT_REG_ADDR(DT_NODELABEL(iomuxc)),
+			      DT_REG_SIZE(DT_NODELABEL(iomuxc)),
+			      MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_NS),
+#endif
+
+	MMU_REGION_DT_COMPAT_FOREACH_FLAT_ENTRY(nxp_mbox_imx_mu,
+			 (MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_NS))
+
+	MMU_REGION_DT_COMPAT_FOREACH_FLAT_ENTRY(nxp_kinetis_lpuart,
+			 (MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_NS))
+
+#if CONFIG_SOF
```

Comment:
We can remove these. They're not defined anywhere in the DTS anyways.

Reply:
Okay.